deploy/roles/eod/tasks/main.yaml

```diff
@@ -49,10 +49,27 @@
   tags:
     - eod
 
+- name: Create admin group
+  group:
+    name: "{{ restricted_group_name }}"
+    state: present
+
 - name: Ensure that EOD log directory exists
   file:
     path: "{{ eod_install_directory }}/log"
     state: directory
+    owner: root
+    group: root
+    mode: 0640
+
+- name: Set recursive admin permissions on log file
+  acl:
+    path: "{{ eod_install_directory }}/log"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rX
+    default: yes
+    state: present
```

Comment:
@tanderegg I am looking at this now - When I first did this there was a reason I didn't have a recursive parameter. I'll keep ya posted...